src/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Balloon Jr.",
+  "short_name": "Balloon Jr.",
+  "orientation": "portrait",
+  "display": "fullscreen",
+  "background_color": "#aeaeae",
+  "theme_color": "#63B1F9",
+  "start_url": "index.html",
```

Comment:
index.html doesn't seem to be cached by sw, so app won't start from homescreen when offline